using-ide-vscode.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: page
+title: Developing Flutter apps in an IDE
+
+permalink: /using-ide-vscode/
+---
+
+<ul class="tabs__top-bar">
+    <li class="tab-link" data-tab-href="/using-ide/">Android Studio / IntelliJ</li>
+    <li class="tab-link current">VS Code</li>
+</ul>
+
+<div class="tabs__content current" markdown="1">
+
+
+The Dart extension provides a fully integrated development experience in the
+Visual Studio Code editor.
+
+* TOC Placeholder
+{:toc}
+
+## Installation and setup
+
+Please follow the [Editor Setup](/get-started/editor/) instructions to install
+the Dart extension.
```

Comment:
perhaps add `; this extension includes Flutter functionality as well`, or similar